testsuite/features/step_definitions/navigation_steps.rb

```diff
@@ -112,12 +112,23 @@
 
 Then(/^I wait until I see the (VNC|spice) graphical console$/) do |type|
   repeat_until_timeout(message: "The #{type} graphical console didn't load") do
-    break unless has_xpath?('.//canvas')
+    break if find(:xpath, '//canvas')
```

Comment:
Take into account it will the default timeout on this find. Probably good, but just saying.

Reply:
that should give the time for the connection to be prepared and the canvas to appear